CHANGELOG.md

```diff
@@ -3,6 +3,106 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.2.0
+
+_Apr 4, 2024_
+
+We'd like to offer a big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Allow higher Data Grid packages' props to be used in MIT (#12365) @MBilalShafi
+- 🚀 Add `setItemExpansion` Tree View API method (#12595) @flaviendelangle
```

Comment:
Anything else we'd want to highlight?
What about the `TreeViewPro`? Maybe we even want to remove its references or even highlight them? 🤔

Reply:
The `@mui/x-tree-view-pro` package is not included in this release, right?
If so, mentioning `TreeViewPro` in the highlights is premature.

Reply:
> Maybe we even want to remove its references

I would update the description of the PR with `(not released yet)` to avoid confusion, and not highlight it